.github/workflows/qe-ocp-414-intrusive.yaml

```diff
@@ -104,6 +104,9 @@ jobs:
           max_attempts: 3
           command: cd ${GITHUB_WORKSPACE}/certsuite-qe; FEATURES=${{matrix.suite}} CERTSUITE_REPO_PATH=${GITHUB_WORKSPACE} USE_BINARY=true JOB_ID=${{github.run_id}} DISABLE_INTRUSIVE_TESTS=true ENABLE_PARALLEL=true ENABLE_FLAKY_RETRY=true make test-features
 
+      - name: Cleanup self-hosted runner images
+        run: docker image prune -f
```

Comment:
It cleans only stopped images. Maybe clean all?
 https://stackoverflow.com/questions/44785585/how-can-i-delete-all-local-docker-images

Reply:
I only want it to clean unused/stopped images.  There might be other github action runs running and actually using live images.